### PR TITLE
Updating repo with changelog.yml

### PR DESCRIPTION
**Pull request by pkesavap**
Signed-off-by: Prajith Kesava Prasad <pkesavap@pkesavap.remote.csb>

### CODE REVIEW

**Comment by pkesavap** [comment]
thanks for the review :-) @felixfontein  i have done  suggested changes .

File: changelogs/.plugin-cache.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+plugins:
```
Comment (felixfontein) [inline changelogs/.plugin-cache.yaml:1]: This file should not be checked in (it's best to put it into `.gitignore`).

Comment (pkesavap) [reply changelogs/.plugin-cache.yaml:1]: just noticed now, will do it.

> This file should not be checked in (it's best to put it into `.gitignore`).

**Comment by pkesavap** [comment]
done, @felixfontein ,
i have added an extra .changelog.yml to safely implement an external automation task. i can remove them if its against the policy.

**Comment by felixfontein** [comment]
@pkesavap there's no policy against that. If it helps you, feel free to add it. Just make sure you don't add unnecessary huge random files, like a venv. (We've seen that in published collections. Not exactly bandwidth friendly :) )

**Comment by pkesavap** [comment]
> @pkesavap there's no policy against that. If it helps you, feel free to add it. Just make sure you don't add unnecessary huge random files, like a venv. (We've seen that in published collections. Not exactly bandwidth friendly :) )

cool, understood :) .

**Pull request merged**